set-web/src/main/java/eu/europeana/set/web/service/impl/UserSetServiceImpl.java

```diff
@@ -374,17 +388,85 @@ public void addNewItemToList(UserSet existingUserSet, int positionInt, String ne
 	}
 
     @Override
-    public UserSet updateUserSetsWithIsDefinedByUrl(UserSet storedUserSet, String apiKey, String action)
+    public UserSet updateUserSetsWithIsDefinedByUrl(UserSet storedUserSet, String apiKey, String action,
```

Comment:
please rename to updateUserSet or updateOpenSet if required

Reply:
isn't this the same functionality as the fetchItems in the controller? Please check if code dupplication was introduced!

Reply:
I believe, there is no code duplication. The second parameter of updateUserSet is apikey. The method fetchItems prepares this parameter and calls updateUserSet method.